UICatalog/Scenarios/ListColumns.cs

```diff
@@ -0,0 +1,332 @@
+					miAlwaysUseNormalColorForVerticalCellLines = new MenuItem ("_AlwaysUseNormalColorForVerticalCellLines", "", () => ToggleAlwaysUseNormalColorForVerticalCellLines ()) { Checked = listColView.Style.AlwaysUseNormalColorForVerticalCellLines, CheckType = MenuItemCheckStyle.Checked },
+					miSmoothScrolling = new MenuItem ("_SmoothHorizontalScrolling", "", () => ToggleSmoothScrolling ()) { Checked = listColView.Style.SmoothHorizontalScrolling, CheckType = MenuItemCheckStyle.Checked },
+					miAlternatingColors = new MenuItem ("Alternating Colors", "", () => ToggleAlternatingColors ()) { CheckType = MenuItemCheckStyle.Checked},
+					miCursor = new MenuItem ("Invert Selected Cell First Character", "", () => ToggleInvertSelectedCellFirstCharacter ()) { Checked = listColView.Style.InvertSelectedCellFirstCharacter,CheckType = MenuItemCheckStyle.Checked},
+				}),
+				new MenuBarItem ("_List", new MenuItem [] {
+					//new MenuItem ("_Hide Headers", "", HideHeaders),
+					miOrientVertical = new MenuItem ("_OrientVertical", "", () => ToggleVerticalOrientation ()) { Checked = listColStyle.VerticalOrientation, CheckType = MenuItemCheckStyle.Checked },
+					miScrollParallel = new MenuItem ("_ScrollParallel", "", () => ToggleScrollParallel ()) { Checked = listColStyle.ScrollParallel, CheckType = MenuItemCheckStyle.Checked },
+					new MenuItem ("Set _Max Cell Width", "", SetListMaxWidth),
+					new MenuItem ("Set Mi_n Cell Width", "", SetListMinWidth),
+				}),
+			});
+
+			Application.Top.Add (menu);
+
+			var statusBar = new StatusBar (new StatusItem [] {
+				new StatusItem(Key.F2, "~F2~ OpenListExample", () => OpenSimpleList (true)),
+				new StatusItem(Key.F3, "~F3~ CloseExample", () => CloseExample ()),
+				new StatusItem(Application.QuitKey, $"{Application.QuitKey} to Quit", () => Quit()),
+			});
+			Application.Top.Add (statusBar);
+
+			Win.Add (listColView);
+
+			var selectedCellLabel = new Label () {
+				X = 0,
+				Y = Pos.Bottom (listColView),
+				Text = "0,0",
+				Width = Dim.Fill (),
+				TextAlignment = TextAlignment.Right
+
+			};
+
+			Win.Add (selectedCellLabel);
+
+			listColView.SelectedCellChanged += (s, e) => { selectedCellLabel.Text = $"{listColView.SelectedRow},{listColView.SelectedColumn}"; };
+			listColView.KeyPress += TableViewKeyPress;
+
+			SetupScrollBar ();
+
+			alternatingColorScheme = new ColorScheme () {
+
+				Disabled = Win.ColorScheme.Disabled,
+				HotFocus = Win.ColorScheme.HotFocus,
+				Focus = Win.ColorScheme.Focus,
+				Normal = Application.Driver.MakeAttribute (Color.White, Color.BrightBlue)
+			};
+
+			// if user clicks the mouse in TableView
+			listColView.MouseClick += (s, e) => {
+
+				listColView.ScreenToCell (e.MouseEvent.X, e.MouseEvent.Y, out int? clickedCol);
+			};
+
+			listColView.AddKeyBinding (Key.Space, Command.ToggleChecked);
+		}
+
+		private void SetupScrollBar ()
+		{
+			var _scrollBar = new ScrollBarView (listColView, true); // (listColView, true, true);
```

Comment:
Method local fields should NOT have the `_` prefix.

Reply:
This was a copy-paste from TableEditor scenario. Should I change both?

Reply:
Sure.